CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.23.1]
+### fixed
+- Link component now is a proper <a> tag so you can right click on it, and will scroll back to top. [#99](https://github.com/plotly/dash-core-components/issues/99)
```

Comment:
Maybe say "Fixes [#99](...), implemented in [#215](https://github.com/plotly/dash-core-components/pull/215/)"

> Link component now is a proper <a> tag. This allows you to right-click on `dcc.Link` and see the standard html link context menu. Also, clicking on a `dcc.Link` will now scroll the next page to the top, rather than keeping the scroll position at the position of the previous page.